src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -22,9 +22,9 @@ static ucs_config_field_t uct_cuda_copy_iface_config_table[] = {
      "Max number of event completions to pick during cuda events polling",
      ucs_offsetof(uct_cuda_copy_iface_config_t, max_poll), UCS_CONFIG_TYPE_UINT},
 
-    {"MAX_EVENTS", "1024",
+    {"MAX_EVENTS", "-1",
```

Comment:
no need to change to int. can use "inf" which will set it to UINT_MAX
https://github.com/openucx/ucx/blob/master/src/ucs/config/parser.c#L127